var/spack/repos/builtin/packages/ncl/package.py

```diff
@@ -143,6 +147,10 @@ def install(self, spec, prefix):
         self.prepare_src_tree()
         make("Everything", parallel=False)
 
+        # Build system may fail without errors, so check for main program.
+        exes = os.listdir(self.spec.prefix.bin)
+        assert "ncl" in exes
```

Comment:
It would be better to raise an exception (maybe `RuntimeError`) with a clear error message